@here/harp-omv-datasource/lib/OmvDecodedTileEmitter.ts

```diff
@@ -289,6 +294,7 @@ export class OmvDecodedTileEmitter implements IOmvEmitter {
                     imageTexture = composeTechniqueTextureName(imageTextureValue, poiTechnique);
                 }
             }
+            meshBuffers.offsetDirection = (env.lookup("offset_direction") as number) ?? 0;
```

Comment:
Here you're ovewriting the offsetDirection set by the previous call to processPointFeature.
Furthermore, this feature is made of multiple points (see loop below). We need to indicate that the offsetDirection applies to all of them. Simplest way is to copy the offsetDirection per point in the feature in the loop below.